adapters/docker_generic/common.php

```diff
@@ -146,8 +147,9 @@ function scp_to_router($src, $dst)
   $sd_ip_addr = $sd->SD_IP_CONFIG;
   $login = $sd->SD_LOGIN_ENTRY;
   $passwd = $sd->SD_PASSWD_ENTRY;
+  $sd_mgt_port = $sd->SD_MANAGEMENT_PORT;
 
-  $ret_scp = exec_local(__FILE__ . ':' . __LINE__, "/opt/sms/bin/sms_scp_transfer -s $src -d $dst_disk/$dst -l $login -a $sd_ip_addr -p '$passwd' ", $output);
+  $ret_scp = exec_local(__FILE__ . ':' . __LINE__, "/opt/sms/bin/sms_scp_transfer -s $src -d $dst_disk/$dst -l $login -a $sd_ip_addr -p '$passwd -P $sd_mgt_port' ", $output);
```

Comment:
`-p '$passwd -P $sd_mgt_port'` ==> `-p '$passwd' -P $sd_mgt_port`
Be carefull to the simple quote around passwd!

Reply:
fix by this commit https://github.com/openmsa/Adapters/pull/295/commits/38a1123dd0717365f493b27ae08b7561d111d3c8